sass/_article.scss

```diff
@@ -0,0 +1,58 @@
+.article {
+  position: relative;
+  margin: 40px 10% 0 10%;
```

Comment:
you don't need the last `10%`, for it is already set by the right side. here, you are overriding `10%` with `10%`